RGPosadas/LitterRally
# TASK-76: View Other User Profiles Frontend

Closes #272 

To test:
* Head on over to http://localhost:4200/other-user-profile
* Change [this line's ID](https://github.com/RGPosadas/Mull/blob/TASK-76/view-other-user-profiles-frontend/apps/mull-ui/src/app/pages/profile/other-user-profile/other-user-profile.tsx#L20) to get the user you want
   * Note that if this ID is the current authenticated user ID, then it will redirect to `/profile`. This is for when a user clicks on their avatar in a messages page, for example. The goal is for /user/user.id to redirect to this page. The whole workflow is to be implemented in #284.
* Ensure that the relationship  between the current user and the other user is accurate with respect to the `Friends` table in the db
